docs/application/web/guides/w3c/perf-opt/performance-improvement.md

```diff
@@ -283,20 +283,19 @@ The basic principle of improving the launch time of a Web application is simply
 - Keep only the first page elements in the `index.html` file.
 
   Take advantage of a useful technique called deferring loading. The UI component creation starts at the DOMContentLoad time, and at this time all the DOM elements in the first page (`index.html`) are constructed. Of course, images and other resources are not yet loaded at this stage.
-  
+
   Often the first page can contain unnecessary elements, and consequently slow down the DOM construction. To avoid the problem, construct the first page to contain only the necessary elements to show the first scene, and put the rest of the pages in another HTML file. Similarly, if you do not need some JavaScript functionality on the first page, load the related JavaScript files only after the first page is loaded.
 
 - Use web frameworks appropriately.
 
   There are many popular web frameworks and they help us create web applications easier. It is possible to use popular web frameworks such as Angular, Vue and so on for Tizen web application development.
 
-  However, if you do not use a framework correctly, it can degrade your application performance. For example, application launch can take a long time. The delay is because most of the web frameworks must be initialized for resource loading while the application starts.
-
-  It is recommended not to use web frameworks if your web application UX is simple and the launch time performance is important.
+  However, if you do not use a framework correctly, it can degrade your application performance on the contrary.
 
-  Alternatively, avoid loading web frameworks until the web application displays the first page. You can create a simple first page that does not use web frameworks. You can also load web frameworks asynchronously while the first page is displayed.
+  For example, application launch can take a long time. The delay is because most of the web frameworks must be initialized for resource loading while the application starts.
+  Therefore it is recommended not to use web frameworks if your web application UX is simple and the launch time performance is important.
```

Comment:
Please add a comma (,) after "Therefore" as mentioned in the previous comment (https://github.com/Samsung/tizen-docs/pull/957#discussion_r353074177).

Therefore, it is recommended not to...